calcom/cal.com
# Fix: Hiding event type loses recurring config

## What does this PR do?

Possible candidate for a hotfix: when hiding an event type that has recurring configuration, it was losing it. This fixes that.

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?

 - Go to Event Type list. 
 - Have one event type with recurring configuration. 
 - Hide the event type
 - It should not lose the recurring configuration
